### PR TITLE
Fabric quarterly report 2024 Q2

### PR DESCRIPTION
**Pull request by denyeart**
Fabric quarterly report 2024 Q2.

### CODE REVIEW

File: project-reports/2024/2024-Q2-Hyperledger-Fabric.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- Identity Mixer algorithm update
+
+# Maintainer Diversity
+
+7 of 9 [maintainers](https://github.com/hyperledger/fabric/blob/main/MAINTAINERS.md) from IBM. A maintainer from Hitachi was added this quarter.
+
+
+# Contributor Diversity
+
+Statistics for Q1 not available... see Insights link below.
```
Comment (denyeart) [inline project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: @ryjones or anyone... maybe I've missed something but I can't get the Insights report to work for Q1.

Comment (ryjones) [reply project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: I have no idea. I'll escalate

Comment (ryjones) [reply project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: @denyeart [this works now](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview/github?project=fabric&repository=all&routedFrom=Github&dateFilters=Last%20Quarter&dateRange=2024-01-01%20to%202024-03-31&compare=PP&granularity=week&hideBots=true)

Comment (denyeart) [reply project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: Ok, this newer Insights dashboard is indeed working.
Too bad the [older Insights dashboard](https://insights.v1.lfx.linuxfoundation.org/projects/hyperledger%2Ffabric/dashboard;subTab=technical?time=%7B%22from%22:%222024-01-01T05:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222024-04-01T04:00:00.000Z%22%7D) doesn't work anymore, it had more detailed information.

**Review by tkuhrt — APPROVED** [review]

**Review by yacovm — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Fabric.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+# Overall Activity in the Past Quarter
+
+The project delivered fixes to the Fabric v2.2 and v2.5 releases. See [release notes](https://github.com/hyperledger/fabric/releases) for details.
+
+The SmartBFT consensus algorithm integration into the Fabric ordering service was completed and delivered in the v3.0.0-beta release in March. The SmartBFT consensus algorithm itself was moved into [hyperledger-labs](https://github.com/hyperledger-labs/SmartBFT).
+
+Also included in the v3.0.0-beta release is a new feature to query all approved chaincodes on a channel.
+
+The last remnants of Azure Pipepines were removed, the transition to Github Actions is now 100% complete, and there was much rejoicing.
```
Comment (non-fungible-nelson) [inline project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: Typo

**Review by non-fungible-nelson — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-Fabric.md (hunk 2 above)
Comment (denyeart) [reply project-reports/2024/2024-Q2-Hyperledger-Fabric.md]: Fixed!

**Review by petermetz — APPROVED** [review]
LGTM

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**